### ISSUE TITLE
AmplifyProvider appears to be broken on latest ui-react@next release.

### ISSUE THREAD

**Issue by alharris-at** [aws-amplify/amplify-ui#688]
Hi, an integration testing app we build just started failing with what looks like an internal error from ui-react.

```
./node_modules/@aws-amplify/ui-react/dist/esm/index.js
Attempted import error: 'Theme' is not exported from '@aws-amplify/ui'.
```

I'm able to reproduce in a clean app with the following steps.

```sh
npx create-react-app testapp \
	--template typescript \
	--use-npm
cd testapp
npm i \
	@aws-amplify/ui-react@next \
	aws-amplify
npm start
```

And pasting the following into `App.tsx`

```tsx
import { AmplifyProvider } from '@aws-amplify/ui-react';

function App() {
  return (<AmplifyProvider>My App</AmplifyProvider>);
}

export default App;
```

My package.json can be found below:

```json
{
  "name": "studioappnew",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@aws-amplify/ui-react": "^0.0.0-next-a9c2969-2021101122330",
    "@testing-library/jest-dom": "^5.15.0",
    "@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "@types/jest": "^26.0.24",
    "@types/node": "^12.20.37",
    "@types/react": "^17.0.34",
    "@types/react-dom": "^17.0.11",
    "aws-amplify": "^4.3.5",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "4.0.3",
    "typescript": "^4.4.4",
    "web-vitals": "^1.1.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```

**Comment by alharris-at** [comment]
Just confirmed the issue was introduced in https://github.com/aws-amplify/amplify-ui/commit/5257a31092b46a10abda99307779592b63d10890, as the `0.0.0-next-248121b-2021101120841` release works, but the `0.0.0-next-5257a31-20211011212613` release fails.

**Comment by dbanksdesign** [comment]
Should be fixed with #692. For some reason when packaging the bundler thinks Theme is a JS object and not a type and tries to export a non existent object

**Comment by dbanksdesign** [comment]
Just verified with a new CRA project and pulling `@aws-amplify/ui-react@next` works!
